humanoid_league_transform/src/humanoid_league_transform/transformer.py

```diff
@@ -44,9 +44,12 @@ def __init__(self):
         cam_info_counter = 0
         while self._camera_info is None:
             rospy.sleep(0.1)
+            cam_info_counter += 1
             if cam_info_counter > 50:
```

Comment:
Now that it works the value could be increased. 5 Seconds seem a little bit short if the robot starts up.

Reply:
I see, I increased the limit to 10 seconds.